utils/Felt.go

```
@@ -31,6 +31,20 @@ func HexToFelt(hex string) (*felt.Felt, error) {
 	return new(felt.Felt).SetString(hex)
 }
 
+// HexToFelt converts a hexadecimal string to a *felt.Felt object, ignoring errors.
+//
+// Note: only use this function if you are sure that the input is a valid felt input.
+// Not recommended for production use. Always handle errors correctly.
+//
+// Parameters:
+// - hex: the input hexadecimal string to be converted.
+// Returns:
+// - *felt.Felt: a *felt.Felt object
+func HexToFeltNoErr(hex string) *felt.Felt {
+	felt, _ := new(felt.Felt).SetString(hex)
+	return felt
+}
```

Give this is only used in tests we could do something like this,
```
func HexToFelt(t testing.TB, hex string) *felt.Felt {
	t.Helper()
	f, err := new(felt.Felt).SetString(hex)
	require.NoError(t, err)
	return f
}
```
In general I would advise against ignoring errors in functions like this

---

Me too.
Also, for tests there's already the `TestHexToFelt` func that does exactly what you mentioned.

I thought about this function when we need to put some values that we have sure are valid felts (like addresses, hashes...) into (for e.g.) an object, and as the felt (for good reasons) returns an error in such operations we need to declare and set the variable previously. With this function we can set it directly. WDYT?

---

Currently this function is only used in tests, and should be replaced with `TestHexToFelt`. 

I see the appeal of having some method to create felts that we know are valid, but imo this should be pushed to a separate PR for now